Ginger/GingerCore/Drivers/SeleniumDriver.cs

```diff
@@ -3688,7 +3690,7 @@ private ElementInfo GetElementInfoWithIWebElementWithXpath(IWebElement el, strin
             EI.ID = GenerateElementID(el);
             EI.Value = GenerateElementValue(el);
             EI.Name = GenerateElementName(el);
-            EI.ElementType = GenerateElementType(el);
+            EI.ElementTypeEnum = GetElementTypeEnum(el);
             EI.Path = path;
```

Comment:
need to have here EI.ElementType = GenerateElementType(el); as well, enum is on top the previous string type